projects/storefrontlib/src/shared/components/anonymous-consents-dialog/anonymous-consent-dialog.component.html

```diff
@@ -1,5 +1,5 @@
 <div
-  class="modal-dialog modal-dialog-centered modal-lg"
+  class="cx-anonymous-consent-dialog"
```

Comment:
cool. Can we get ride of all notions of modal? 

modal-content
modal-header
modal-title
closeModal()
modal-body